client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -170,7 +170,7 @@ export default React.createClass( {
 				text={ i18n.translate( 'A new version is available.' ) }
 				status="is-warning"
 				button={ i18n.translate( 'Update to %(newPluginVersion)s', { args: { newPluginVersion: newVersion } } ) }
-				onClick={ this.handlePluginUpdates }
+				onDismissClick={ this.handlePluginUpdates }
```

Comment:
hmm, I don't understand why, but this case is not working:

![image](https://cloud.githubusercontent.com/assets/1554855/11506080/c42b7c94-984d-11e5-8abd-21248ab50487.png)